Source/WebCore/accessibility/AccessibilitySVGObject.cpp

```diff
@@ -319,4 +327,59 @@ AccessibilityRole AccessibilitySVGObject::determineAccessibilityRole()
     return AccessibilityRenderObject::determineAccessibilityRole();
 }
 
+void AccessibilitySVGObject::setParent(AccessibilityRenderObject* parent)
+{
+    m_parent = parent;
+}
+
+AccessibilityObject* AccessibilitySVGObject::parentObject() const
+{
+    // If a parent was set because this is a remote SVG resource, use that
+    // but otherwise, we should rely on the standard render tree for the parent.
+    if (m_parent)
+        return m_parent.get();
+
+    return AccessibilityRenderObject::parentObject();
+}
+
+bool AccessibilitySVGObject::hasAccessibleContent() const
```

Comment:
Maybe this method name should change now that it's on the more general `AccessibilitySVGObject` class. Maybe `isRootWithAccessibleContent()`?